### ISSUE TITLE
Update GCC version of m2w64-toolchain

### ISSUE THREAD

**Issue by twiecki** [conda-forge/toolchain-feedstock#60]
It seems like the gcc version installed for `m2w64-toolchain` on windows is quite old (version 5.3.0). More recent versions are 8.3.0. Is it possible to update them?

This is causing problems for compilation under windows for pymc and aesara: https://github.com/pymc-devs/pymc/issues/4937

**Comment by twiecki** [comment]
@conda-forge/core is no one maintaining the conga-forge compiler package?

**Comment by xhochy** [comment]
No, there is nobody currently actively maintaining the `m2w64` stack on conda-forge. We build all Python packages using MSVC though by default. 

It would be nice if someone would step to do so but for `pymc` it would probably be simpler to migrate to MSVC or clang on Windows.

**Comment by mwtoews** [comment]
Is this the archived recipe https://github.com/conda-archive/conda-recipes/blob/master/msys2/m2w64-toolchain/meta.yaml ? Is it a matter of forking it to conda-forge, and updating the links, running CI checks and patching?

**Comment by twiecki** [comment]
@mwtoews Yes, I think that's what's involved. It would be a major contribution, not having up-to-date windows compilers in conda-forge is a big drawback.

**Comment by jaimergp** [comment]
I think you are looking for this "feedstock": https://github.com/conda-forge/msys2-recipes/tree/master/msys2

**Comment by mwtoews** [comment]
Note that I've posted a discussion https://github.com/msys2/MINGW-packages/discussions/12013 seeking volunteers to help. Recipes are best maintained by communities that develop the components.
